index.html

```diff
@@ -0,0 +1,52 @@
+<link rel="stylesheet" href="css/style.css">
+<link href="https://fonts.googleapis.com/css?family=Poppins:200i,300,400&amp;display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+  <nav id="navbar">
+    
+       <a href="#welcome-section">About</a>
+       <a href="#projects">Work</a>
+       <a href="#conact">Contact</a>
+    
+
+  </nav>
+  
+<script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+<div id="welcome-section">
+  <h1>Hey I am Mimic</h1>
+  <h4 id="head4">a web developer</h4>
+</div>
+
+<div id="projects">
+  <div>
+    <h2 class="project-title">These are some of my projects</h2>
+  </div>
+  <div id="section">
+    <div id="random">
+      <a href="https://codepen.io/freeCodeCamp/full/qRZeGZ" target="_blank">
+        <div>
+          <img class="image" src="images/random-quote-machine.png" alt="img">
+          <p class="project-para">
+            <span class="code">&lt;</span>
+            Random Quote Machine
+            <span class="code">/&gt;</span>
+          </p>
+        </div></a>
+    </div>
+  <div id="first-link">
+    <a href="https://codepen.io/freeCodeCamp/full/zNqgVx"> 
+      <img class="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg" alt="project1">
+      <p class="project-name">
+        <span class="code">&lt;</span>
+        Tribute Page 
+        <span class="code">&#47;&gt;</span>
+      </p>
+    </a>
+  </div>
+</div>
```

Comment:
I think would be better here If you use `<script>` to warp projects instead of `div`, and for each project, you can warp each project with `<article>` also instead of `div`.

These two changes are for more semantical reasons. [more](https://www.w3schools.com/html/html5_syntax.asp)
And check [this](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Document_and_website_structure) in order to have better ideas about How to structure your page properly?